app/controllers/concerns/content_items_concern.rb

```diff
@@ -0,0 +1,19 @@
+module ContentItemsConcern
```

Comment:
A few comments here:

1. I don’t think this is a fat controller for sorting the content items. Indeed the code is something like:

```ruby
   @content_items.order(params[:sort] => params[:order])
```
This does not look to me like a fat controller for our first test.

2. If we extract a concern for sorting, it does not need to be related to ContentItems, and in my opinion, it does not need to be related to a controller. It feels more close to a model, or to a QueryObject, in case you want to follow that path.